File: sorting/bubble_sort/bubble_sort.py
x = [int(i) for i in input().split()]
# enter numbers with spaces
# Ex: 1 2 3 4 5
# x = [1,2,3,4,5]
print("Original array:")
print(x)
# Ascending order
for i in range(len(x)-1,-1,-1):
    for j in range(i):
        if(x[j]>x[j+1]):
            x[j],x[j+1] = x[j+1],x[j]
print("Sorted array(ascending):")
print(x)

File: sorting/bubble_sort/bubblesort.cpp
#include <iostream>
#include <vector>
using namespace std;
void bubblesort(vector<int>);
int main()
{
    int n;
    cout << "Enter the size of the array" << endl;
    cin >> n;
    vector<int> arr; //vector used like array
    int i=0;
    int num; //number for dynamic array
    cout << "Enter the elements of the array" << endl;
    for(i=0;i<n;i++)
    {
        cin >> num;
        arr.push_back(num);
    }
    bubblesort(arr);
}
void bubblesort(vector<int> array)
{
    cout << "BUBBLE SORTED ARRAY: ";
    //doing bubblesort for doing ascending order
    for(int i=array.size()-1;i>0;i--) //loop for decreasing array size
    {
        for(int j=0;j<i;j++) //loop for 
        {
            if(array[j] > array[j+1])
            {
                //swapp
                int temp = array[i];
                array[i] = array[j];
                array[j] = temp;
            }
        }
    }
    //output the array
    for(int i=0;i<array.size();i++)
    {
        cout << array[i] << " ";
    }
    cout << endl;
}